const generalErrorMessage = 'エラーが発生しました。';
